lib/beacon_web/hooks/assign_agent.ex

```diff
@@ -0,0 +1,14 @@
+defmodule BeaconWeb.Hooks.AssignAgent do
+  import Phoenix.Component
+
+  alias Beacon.Authorization
+
+  def on_mount(:default, _params, session, socket) do
+    authorization_bootstrap = Map.get(session, "authorization_bootstrap", nil)
```

Comment:
I'm assuming you'll be passing `authorization_bootstrap` from the client, right? Which means this value is a public contract and users should know about it. Can we discuss about it when you have some time?